pylint/pyreverse/writer.py

```diff
@@ -140,12 +158,14 @@ def get_values(self, obj):
                 )
 
                 if func.args.args:
-                    args = [arg for arg in func.args.args if arg.name != "self"]
+                    argument_list = [
```

Comment:
```suggestion
                    arguments: List[str] = [
```
Personal preference but when naming an iterable I like a plural word so I can do ``for argument in arguments`` later on.

Reply:
I agree, changed it.